data/test/macros/wml_unit_test_macros.cfg

```diff
@@ -113,13 +165,13 @@ Elvish Archer#endarg
 #enddef
 
 #define FAIL_IF_NOT FLAG NOT_EQUALS
-[if]
-    [variable]
-        name={FLAG}
-        not_equals={NOT_EQUALS}
-    [/variable]
-    [then]
-        {FAIL}
-    [/then]
-[/if]
+    [if]
```

Comment:
Why randomly indenting stuff? (Or was this caused by `wmlindent`? In which case the question would be, why isn't the change already in master and why didn't the CI fail because it's not?)

Reply:
It was caused by `wmlindent`. There's a CI exclusion in 599504cee3544ad19007913aae35b81b5375b6e7, for which the GNA bug became Github's #1397.

Reply:
Revised plan - I won't change `wml_unit_test_macros.cfg` in this PR.

Instead of adding the `COMMON_KEEP_A_B_UNIT_TEST` to `wml_unit_test_macros.cfg`, I want to put it in a separate file. There will then be a [separate PR](https://github.com/wesnoth/wesnoth/pull/6627) for master (possibly backported too) that moves `GENERIC_UNIT_TEST` to its own file and extracts `COMMON_KEEP_A_B_C_D_UNIT_TEST` from the 4 side setup in `firststrike_and_laststrike`.

The main reason for having these in separate files is Git's fuzzy-patch matching when merging and rebasing. Having large blocks of identical text in `wml_unit_test_macros.cfg` can mean the wrong section gets patched.